src/main/java/com/salesforce/cdp/queryservice/util/Constants.java

```diff
@@ -34,18 +34,10 @@ public class Constants {
     public static final String DRIVER_NAME = "QueryService-jdbc";
     public static final String DRIVER_VERSION = "1.0";
 
-    // Common client id and secret information
+    // Common server information
     public static final String PROD_SERVER_URL = ".salesforce.com";
```

Comment:
We could remove the URL as well ryt?

Reply:
No these constants are being used in other code snippets as well.

Reply:
ok